subprojects/docs/src/docs/release/notes.md

```diff
@@ -32,6 +32,20 @@ details of 2
+
+The impact on your build may vary, but we are happy to ship this improvement that will noticeably shorten your feedback loop when editing build logic.
```

Comment:
We should focus on the readers rather than ourselves - for example say "you will be happy to see this improvement" rather than "we are happy to ship this improvement".

Another proposal:
"While the impact on your build may vary, most builds can expect a noticeable shortening of build editing feedback loop with Kotlin DSL thanks to this improvement"